homeassistant/components/linkplay/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "domain": "linkplay",
+  "name": "LinkPlay",
+  "after_dependencies": ["http"],
+  "codeowners": ["@Velleman"],
+  "config_flow": true,
+  "dependencies": ["ssdp", "zeroconf"],
+  "documentation": "https://www.home-assistant.io/integrations/linkplay",
+  "integration_type": "hub",
+  "iot_class": "local_polling",
+  "requirements": ["python-linkplay==0.0.5"],
```

Comment:
We at Velleman use release branches (e.g. https://github.com/Velleman/python-linkplay/tree/releases/0.0.5) to organize releases. We'll add GitHub releases from these branches so that the users of the library can immediately see the releases.